packages/frontend/src/components/Layout/index.tsx

```diff
@@ -19,22 +18,19 @@ type PublicLayoutProps = {
 
 const drawerLinks = [
   {
-    Icon: SchemaIcon,
-    primary: 'drawer.flows',
+    Icon: PipeIcon,
```

Comment:
TBH i think this is just a nit, I wouldn't mind using `createIcon`. But we can avoid it if you do this:

```
import pipeIcon from 'assets/pipeicon_interactiondefault.svg'
import { Image } from '@chakra-ui/react'

const drawerLinks = [
  {
    Icon: () => <Image src={pipeIcon} />
    // ...
```

Reply:
oh I tried this initially but realised that I can't change the svg content (for hover and active) because images dont allow me to do so